compiler/compile/Compilation.hpp

```diff
@@ -49,7 +49,8 @@ class OMR_EXTENSIBLE Compilation : public OMR::CompilationConnector
          TR::Options &options,
          TR::Region &heapMemoryRegion,
          TR_Memory *memory,
-         TR_OptimizationPlan *optimizationPlan) :
+         TR_OptimizationPlan *optimizationPlan,
+         bool shouldCompile = true) :
```

Comment:
Would you please also remove this `shouldCompile` support from the pull request? With the listener approach, I don't *think* it will be needed (which is another advantage of that approach).